conda/recipes/blazingsql/meta.yaml

```diff
@@ -28,6 +28,7 @@ requirements:
         - python
         - pip
         - setuptools
+        - spdlog >=1.7.0,<1.8.0
```

Comment:
Why we need to declare spdlog on here? Is already declared in the rmm package
https://github.com/rapidsai/rmm/blob/branch-0.16/conda/recipes/librmm/meta.yaml#L31
We should avoid to include here transitive dependencies cc @mario21ic

Reply:
without it, its picking the latest spdlog 1.8.0 which we dont want.Not sure why the rmm one is not being used